src/components/tables/Store.ts

```diff
@@ -0,0 +1,123 @@
+import { LiveSpecsExtQuery } from 'hooks/useLiveSpecsExt';
+import produce from 'immer';
+import { devtoolsOptions } from 'utils/store-utils';
+import create, { GetState } from 'zustand';
+import { devtools, NamedSet } from 'zustand/middleware';
+
+export interface SelectableTableStore {
```

Comment:
Table store that houses all the rows (including their details so actions can be taken on them) as well as a map of selected rows. Keeping these as two separate maps to keep things easier to check... like `selected.has(key)`